tests/features/collection/propose.feature

```diff
@@ -135,3 +135,27 @@ Feature: Proposing a collection
     When I click on element ".js-form-item-field-ar-closed-value label"
     Then the option "Only members can create new content." should be selected
     And the option "Only collection facilitators can create new content." should not be selected
+
+  @javascript
+  Scenario: Propose collection form fields should be organized in groups.
+    Given I am logged in as an "authenticated user"
+    # The browser windows has to be wider than 640px, which is the default
```

Comment:
The focus point is that the script won't apply for viewports < 640, but only the default collapse behaviour of HTML5 details element. So everything will be rendered for sure. And we have other non-js test to verify that all the fields are in place.